core/server/influx-metrics.js

```diff
@@ -15,21 +13,19 @@ module.exports = class InfluxMetrics {
   }
 
   async sendMetrics() {
-    const auth = {
-      user: this._config.username,
-      pass: this._config.password,
-    }
     const request = {
-      uri: this._config.url,
+      url: this._config.url,
       headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
       body: await this.metrics(),
       timeout: this._config.timeoutMillseconds,
-      auth,
+      username: this._config.username,
+      password: this._config.password,
+      throwHttpErrors: false,
```

Comment:
this just prevents `got` from throwing on some non-successful response status code right?

Reply:
Yep. By default got throws an exception if it gets a 404, 500 or whatever. This tells it to return a response. I mainly did this because the existing error handling code assumes this behaviour from request, but I could have rewritten the error handling code here

https://github.com/badges/shields/blob/fd7eddc7bb406b272b8bbf4155315c5f3b198a03/core/server/influx-metrics.js#L33-L45

to handle both cases in the catch block instead. I don't think it makes much difference either way though.